lib/asciidoctor/csa/csa.rng

```diff
@@ -79,8 +79,13 @@
       </element>
     </define>
   </include>
-  <define name="csand-standard">
-    <element name="csand-standard">
+  <define name="csa-standard">
+    <element>
+      <choice>
+        <!-- https://stackoverflow.com/a/49798676/902217 -->
+        <name>csand-standard</name>
+        <name>csa-standard</name>
+      </choice>
```

Comment:
@ronaldtse @opoudjis we support both names, no backward compatibility tests added yet, but I will try to add someting

Reply:
The gem isn't in popular use, let's change it without backward compatibility.